mlos_bench/mlos_bench/tests/tunable_groups_fixtures.py

```diff
@@ -60,7 +60,8 @@
                 "type": "int",
                 "default": 2000000,
                 "range": [0, 1000000000],
-                "log": false
+                "log": false,
+                "quantization": 100
```

Comment:
Makes sure we test quantized values with our default tunable groups fixtures.
The change was actually quite small, so I included it here as well, but I can split it back out again instead too.

Reply:
Moved that over to #814